### DISCUSSION TITLE
Grid in ANPHON (KPMODE = 2)

### DISCUSSION THREAD

**Discussion by RicardoCecconello** [ttadano/alamode#166] [Q&A] [answered]
Dear Dr. Tadano,

The k grid used in ANPHON (with `KPMODE = 2`) should be commensurate with the supercell size (i.e., if I use a 4x4x2 supercell, this grid should be 40 40 20 or multiples), or should be commensurate with the dimensions of the primitive cell?

Thanks in advance,
Ricardo.

**Comment by ttadano** [comment, marked as answer]
Hi,

For the `&kpoint` field, the k-point grid does not need to be commensurate with the supercell size, unlike the case of `KMESH_INTERPOLATE` in the `&scph` field.

However, it is recommended and sometimes necessary (for thermal conductivity calculations) to use a k-point grid that does not break the symmetry of the input lattice. For example, when your supercell is cubic, you should avoid using anisotropic k-point sampling such as `10 10 20`.

**Reply by RicardoCecconello** [reply]
Thank you so much!
